### PR TITLE
chore: [MH-000] storybook updates

### PR DESCRIPTION
**Pull request by EvgeniyKvasyuk**

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "@storybook/addon-interactions"
+  ],
+  features: {
+    emotionAlias: false,
+  },
+  "framework": "@storybook/react",
+  webpackFinal: async (config) => {
+    config.resolve.alias = {
+      ...config.resolve.alias,
+      '@common': path.resolve(__dirname, "../src/common"),
```
Comment (VladSolodjankin) [inline .storybook/main.js:35]: we don't have ../src/common directory

File: .storybook/preview.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import {StyledEngineProvider} from '@mui/material/styles';
```
Comment (VladSolodjankin) [inline .storybook/preview.js:1]: can we setup eslint for storybook?

Comment (EvgeniyKvasyuk) [reply .storybook/preview.js:1]: I think yes. But I would do it in the next prs

**Review by VladSolodjankin — APPROVED** [review]

**Pull request merged**